### PR TITLE
chore: update exchange schema

### PR DESCRIPTION
**Pull request by fladson**
follow up work from https://github.com/artsy/exchange/pull/1171 and https://github.com/artsy/exchange/pull/1182

### CODE REVIEW

**Comment by erikdstock** [comment]
> Input field 'bankAccountId' was removed from input object type 'CommerceSetPaymentInput'

Why remove `bankAccountId`?

**Comment by fladson** [comment]
> > Input field 'bankAccountId' was removed from input object type 'CommerceSetPaymentInput'
> 
> Why remove `bankAccountId`?

Please ignore this PR for now, we should merge this one instead https://github.com/artsy/metaphysics/pull/4098/files until the deprecation is addressed.

**Review by kiryl-zubarau — DISMISSED** [review]

File: src/schema/v1/ecommerce/types/enums/payment_method_enum.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { GraphQLEnumType } from "graphql"
+
+export const PaymentMethodEnum = new GraphQLEnumType({
```
Comment (kiryl-zubarau) [inline src/schema/v1/ecommerce/types/enums/payment_method_enum.ts:3]: Are we really need this enum?

Comment (fladson) [reply src/schema/v1/ecommerce/types/enums/payment_method_enum.ts:3]: Do you know another way to define it? I followed the project convention for enums.

Comment (starsirius) [reply src/schema/v1/ecommerce/types/enums/payment_method_enum.ts:3]: When Exchange schema is merged into Metaphysics, we [rename and prefix](https://github.com/artsy/metaphysics/blob/ca3ca865c173d956c551a60db9d70d371c75cd9e/src/lib/stitching/exchange/schema.ts#L27-L29) the types with `Commerce...`. So the Exchange payment method enum is [already in MP](https://github.com/artsy/metaphysics/blob/ca3ca865c173d956c551a60db9d70d371c75cd9e/_schemaV2.graphql#L4710-L4725) (for free!) as `CommercePaymentMethodEnum` and that's what we'll use for MP. So I think this addition is not needed.

Comment (fladson) [reply src/schema/v1/ecommerce/types/enums/payment_method_enum.ts:3]: good to know, but what about v1? I think that is needed for v1

Comment (starsirius) [reply src/schema/v1/ecommerce/types/enums/payment_method_enum.ts:3]: We want to deprecate v1 and I believe we have migrated all BNMO related code to use v2. So I think we can ignore updating v1 schema here (which can be useful speeding up the deprecation). If I'm wrong, we'll find out! 😛

Comment (fladson) [reply src/schema/v1/ecommerce/types/enums/payment_method_enum.ts]: Do you mean omitting `_schema` changes?

File: src/schema/v1/ecommerce/set_order_payment_mutation.ts [hunk 2]
```diff
@@ -47,7 +47,8 @@ export const SetOrderPaymentMutation = mutationWithClientMutationId<
       mutation setOrderPayment($orderId: ID!, $creditCardId: String!) {
         ecommerceSetPayment(input: {
           id: $orderId,
-          creditCardId: $creditCardId,
+          paymentMethod: CREDIT_CARD,
+          paymentMethodId: $creditCardId,
```
Comment (starsirius) [inline src/schema/v1/ecommerce/set_order_payment_mutation.ts:51]: Updating the implementation of v1 so the tests pass with this breaking change. This is a good reminder that we should see where we are at with deprecating/retiring v1.

**Pull request merged**